.github/workflows/test_e2e_web_apps.yaml

```diff
@@ -78,35 +65,92 @@ jobs:
         working-directory: ./contracts/vlayer
         run: forge soldeer install
 
-      - name: Run e2e tests
+      - name: Create reusable Cargo target dir
+        id: cargo_target
+        uses: ./.github/actions/reuse-cargo-target
+        with:
+          targets_path: ${{ vars.VLAYER_CARGO_TARGET_DIRS }}
+          target_name: test-e2e-bonsai-opsepolia
+      - name: Build binaries
         env:
+          CARGO_TARGET_DIR: ${{ steps.cargo_target.outputs.CARGO_TARGET_DIR }}
+          RISC0_USE_DOCKER: 1
           RUSTC_WRAPPER: ${{ steps.rust_pre.outputs.RUSTC_WRAPPER }}
-          PROVING_MODE: "dev"
-          VLAYER_ENV: dev
-          VLAYER_TMP_DIR: ./artifacts
-          PLAYWRIGHT_TEST_X_COM_AUTH_TOKEN: ${{ secrets.PLAYWRIGHT_TEST_X_COM_AUTH_TOKEN }}
-        run: xvfb-run bash/e2e-web-apps-tests.sh
```

Comment:
Is this `sh` script no longer used and can be removed?
If so, let's remove it in a separate PR - and also check the CI this way.